### DISCUSSION TITLE
Not blocking Followed Accounts’ followed accounts and other suggestions

### DISCUSSION THREAD

**Discussion by Ktulos** [kheina-com/Blue-Blocker#298] [Ideas]
Apologies if these are not possible, or is already in the works. If you’re following someone who is following someone else who has a blue checkmark, would it be possible to have an option to not block that person? It feels tedious to try and go through someone’s Following list and try and white list all of the accounts that they are following to prevent them from not showing up on one’s timeline/For You page if they get retweeted and are not big enough to not get automatically blocked depending on one’s settings.

An option to not block blue checks being Quote-Tweeted would also be a good option, but no idea if this is viable. Seeing what people are discussing about someone with a blue check would be appreciated since not everyone screenshots the Blue users still.

Another option to not block Blue check users in the same community you are in. I wasn’t following the two mods/admins and the result is that I got booted when they got blocked and I did not catch it. I was able to re-join after unblocking them, but it not being an issue in the first place would be handy.

I appreciate your consideration.

**Comment by rougetimelord** [comment]
1. See #122
2. Pretty easy to implement, we pull the quoted tweet in the middle of tweet parsing. Could add a setting to not do that.
3. ATP the extension doesn't touch communities at all, see #245. I'm not sure if there is a way to get a list of all community users without sending an additional API request (which is not within the scope of the extension) and/or if the API tells the client whether the user shares a community with another account.

**Reply by Ktulos** [reply]
Ah, thanks for the prompt and succinct response. I’m not too familiar with Github yet and wasn’t aware of the Issues tab. I appreciate all the work going into this add-on that everyone here is putting in.
